charts/argo-cd/values.yaml

```diff
@@ -582,7 +582,10 @@ controller:
     successThreshold: 1
     # -- Number of seconds after which the [probe] times out
     timeoutSeconds: 1
+
   livenessProbe:
+    # -- Enable Kubernetes liveness probe for Application controller
+    enabled: false
```

Comment:
@mkilchhofer @jmeridth Do you want to keep this optional or remove it completely. I agree that this probe might be dangerous under load as it can lead to crash looping.

Reply:
I would remove it. As an alternative we could extend the description text with a bold warning maybe? :)

Reply:
Removed as requested.